client/jetpack-connect/test/__snapshots__/signup.js.snap

```diff
@@ -213,8 +213,7 @@ exports[`JetpackSignup should render 1`] = `
                 .
               </p>
               <button
-                class="button signup-form__submit form-button is-primary"
-                locale="en"
```

Comment:
I cannot find any code which depends on this non-standard attribute. I'd expect if it was used it would be data-locale. [It's passed to all FormButton components via the localize higher order component](https://github.com/Automattic/wp-calypso/blob/b159deb4ada78061bb88437f5d7e50e460251c4a/client/components/forms/form-button/index.jsx#L38), and is removed now that we're using a core Button.